_data/site_menu.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
-installation:
-  title: Installation
+installing:
+  title: Installing
```

Comment:
As an aside... See line 10 and blow... Kubernetes is the only container based installation with the rest being appliances.  I wonder if there should be categories for installing on container/podified and appliances.

Reply:
Good idea in my opinion.

Reply:
Added commit for this.